app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -0,0 +1,416 @@
+<%
+  title ||= "GOV.UK - The best place to find government services and information"
+  html_lang ||= "en"
+%>
+
+<!DOCTYPE html>
+  <!--[if lt IE 9]><html class="lte-ie8" lang="<%= html_lang %>"><![endif]-->
+  <!--[if gt IE 8]><!--><html lang="<%= html_lang %>"><!--<![endif]-->
+  <head>
+    <meta charset="utf-8" />
+    <title><%= title %></title>
+
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
Does anyone know what the `data-turbolinks-track` attribute does and if it's still needed?

Reply:
As far as I can see none of the public-facing application is using turbolinks. I suggest we remove this unless we find evidence of it being used.

Reply:
Based on @alex-ju suggestion, I have removed the reference to `turbolinks` as I could not find any evidence that it was doing anything.